apm-collector/apm-collector-stream/src/main/java/org/skywalking/apm/collector/stream/worker/util/Const.java

```diff
@@ -1,13 +1,11 @@
-package org.skywalking.apm.collector.agentstream.worker;
+package org.skywalking.apm.collector.stream.worker.util;
 
 /**
  * @author pengys5
  */
 public class Const {
     public static final String ID_SPLIT = "..-..";
     public static final String IDS_SPLIT = "\\.\\.-\\.\\.";
```

Comment:
In analysis process, just append ID_SPLIT.  The ui will split the aggregate id. So the operate  has no effect on performance.